data.md

```diff
@@ -0,0 +1,142 @@
+## Réglementation des données
+### :small_red_triangle: Au niveau français
+##### *Loi Informatique et Liberté*
+Les dispositions de cette loi portent sur  le traitement de données automatisé ou pas (cette seconde partie étant bien souvent oubliée).
+Elle oblige différentes choses :
+* Déclarer les traitement qui sont effectués.
+* Assurer la sécurité des données en évaluant les risques liés aux traitements et en metant en place des solutions adaptés ainsi qu'en obligeant le respect de la confidentialité des données.
+* Conserver les données personnelles dans l'Union Européenne. 
+* Informer les personnes sur les différents traitement effectués.
+* Ne pas traiter les données sensibles telles que celle relevant de la santé, religion ...
+* Le traitement doit avoir un but spécifique et la collecte de données doit se faire de manière pertinante et avec le consentement des personnes.
+* Le respect des droits des personnes à l'accés et à la communication des données, à l'opposition au traitement et à la modification/rectification.
+
+###### source : [CNIL](https://www.cnil.fr/fr/loi-78-17-du-6-janvier-1978-modifiee)
+
+### :small_red_triangle: Au niveau Européen
+##### *La Réglementation Générale de la Protection des Données*
+Cette réglementation est applicable à partir du 26 mai 2018. Il élargie certains droits des citoyens de l'Union Européenne.
+* **Le droit de consentement** : les données ne peuvent être collectées sans un accord explicite et positif de la personne.
+* **Le droit de portablilité** : la personne doit pouvoir récupérer ces données pour les transférer ailleur.
+* **Le droit d'accès et de rectification** : la personne doit pouvoir avoir accès à ces données et les modifier.
+* **Le droit de transparence** : droit de savoir à quoi servent les données.
+* **La minimisation** : les données utilisées doivent être pertinantes avec le but du traitement.
+* **Le droit d'oppoition** : droit se s'opposer au traitement de ces données à tout moment.
+* **Le droit à l'oubli** : la personne doit pouvoir supprimer ces données et la conservations de celles-ci est limitée.
+* **Le droit de profilage** : le droit de ne pas faire l'objet d'une décision fondée sur un traitement de données automatisé.
+* **Le droit de sécurité** : la personne doit avoir le droit d'avoir ces données constamment protégés.
+* **Le droit de notification** : la personne doit être informée en cas de fuite de données.
+
+###### source : [CNIL](https://www.cnil.fr/fr/reglement-europeen-protection-donnees)
+
+## Les outils d'analyse de données
+
+
+* tools to analise data
+http://asq.org/learn-about-quality/data-collection-analysis-tools/overview/overview.html
+https://www.coursera.org/learn/data-analysis-tools
```

Comment:
that's not a tool, but a course about tools.

Please don't provide links without explanation,